README.md

```diff
@@ -78,6 +78,12 @@ If after installing OWIE into your board it reports that your battery is at 1% e
 This problem occurs because the BMS goes through a state reset and doesn't know the status of the battery, and plugging the board
 into a charger corrects this issue by forcing the BMS (and controller potentially) to do a state check.
 
+### Forgotten or mistaken password
+
+With having a secured wifi network generated by the board, we figured it could cause some trouble if you forgot that password and wanted to update your board.
+Therefore if you put the board into recovery mode by power-cycling (rebooting) it 3 times in 10 seconds, it will reset all settings.
+See Recovering an Owie flash below for more information on the behavior when Owie enters recovery mode.
+You don't even need to connect to the recovery network, just waiting another 5-10 seconds (so the rolling restart window is empty) and power cycling your board again will reset everything.
```

Comment:
This part ain't true. Rolling restart window doesn't apply to recovery mode. Remove pls.